runtime/parachains/src/paras.rs

```diff
@@ -279,91 +262,45 @@ pub struct ParaGenesisArgs {
 	pub parachain: bool,
 }
 
-decl_storage! {
-	trait Store for Module<T: Config> as Paras {
```

Comment:
Pallet must always be imported as `Paras` for this to be backwards compatible

Reply:
Kusama, Westend, and Rococo look okay

Reply:
A re-export should be fine here, i.e. `pub use pallet::Pallet as Paras`